pyqt-apps/siriushla/as_ap_launcher/main.py

```diff
@@ -160,6 +159,18 @@ def _setupUi(self):
             parent=self,
             init_channel=self._prefix+evg_name+':STATEMACHINE',
             color_list=color_list)
+        evg_injection_injcnt = PyDMLabel(self)
+        evg_injection_injcnt.setToolTip(
+            'Count injection pulses when Egun Trigger is enabled.')
+        evg_injection_injcnt.channel = \
+            self._prefix+'AS-Glob:AP-CurrInfo:InjCount-Mon'
+        evg_injection_injcnt.setStyleSheet(
+            'QLabel{max-width: 3.5em;}')
+
+        evg_injection_sts_lay = QHBoxLayout()
+        evg_injection_sts_lay.setSpacing(3)
```

Comment:
@anacso17, wouldn't it be better to put a `stretch` between the widgets instead of a `space`?

Reply:
well pointed, done!